developer/webservices/configuring-role-based-authorization.md

```diff
@@ -1,104 +1,107 @@
+         DomainName (Optional): Domain for the user. For local machine account either do not include them or give the machine name. Do not give empty string
+         MapIncomingUser (Optional): Boolean value indicating whether to execute cmdlet in the context of network client.
+
+         User credentials and MapIncomingUser=true are exclusive.
+       -->
+       <Group Name="NonAdminGroup" MapIncomingUser="true">
+         <Cmdlets>
+           <Cmdlet>Get-Service</Cmdlet>
+           <Cmdlet>Set-Service</Cmdlet>
+           <Cmdlet>Get-Process</Cmdlet>
+           <Cmdlet>Get-Item</Cmdlet>
+           <Cmdlet>New-Item</Cmdlet>
+           <Cmdlet>Get-Command</Cmdlet>
+           <Cmdlet>ConvertTo-Xml</Cmdlet>
+           <Cmdlet>ConvertTo-Json</Cmdlet>
+           <Cmdlet>ConvertFrom-Json</Cmdlet>
+         </Cmdlets>
+       </Group>
+       <Group Name="AdminGroup" MapIncomingUser="true">
+         <Cmdlets>
+           <Cmdlet>Get-Service</Cmdlet>
+           <Cmdlet>Get-Process</Cmdlet>
+           <Cmdlet>Get-Item</Cmdlet>
+           <Cmdlet>New-Item</Cmdlet>
+           <Cmdlet>Get-Command</Cmdlet>
+           <Cmdlet>ConvertTo-Xml</Cmdlet>
+           <Cmdlet>ConvertTo-Json</Cmdlet>
+           <Cmdlet>ConvertFrom-Json</Cmdlet>
+         </Cmdlets>
+         <Modules>
+           <Module>C:\Windows\System32\WindowsPowerShell\v1.0\Modules\ServerManager\ServerManager.psd1</Module>
+         </Modules>
+       </Group>
+     </Groups>
+     <Users>
+       <!-- User consists of the following :
+         Name: Name of the user. If a user is from a cer
+         AuthenticationType: Authentication type used.
+         DomainName (Optional): Domain for the user
+       -->
+       <User Name="localNonAdmin" AuthenticationType="Basic" GroupName="NonAdminGroup" />
+       <User Name="localAdmin" AuthenticationType="Basic" GroupName="AdminGroup" />
+     </Users>
+   </RbacConfiguration>
+   ```
+
+2. The file contains two `Group` nodes. These represent the two roles used in this example, the `NonAdminGroup` and the `AdminGroup` roles.
+
+   Directly after the closing `Cmdlets` tag in the first `Group` node, add the following XML:
+
+   ```xml
+   <Modules>
+           <Module>C:\Windows\System32\WindowsPowerShell\v1.0\Modules\ServerManager\ServerManager.psd1</Module>
+         </Modules>
+   ```
+
+#### Adding a User to a Group Node
+
+1. Open the **RBacConfiguration.xml** file in a text editor. This file is located in the folder C:\\\inetpub\wwwroot\Modata  if you did not change the endpoint name before installation.
+
+2. Directly after the closing tag in the `Users` node, add the following XML:
+
+   ```xml
+   <User Name="UserName" GroupName="AdminGroup" AuthenticationType="Basic" DomainName="DomainName"/>
+   ```
+
+3. Directly after the XML added in the previous step, add the following XML:
+
+   ```xml
+   <User Name="UserName" GroupName="NonAdminGroup" AuthenticationType="Basic" DomainName="DomainName"/>
+   ```


```

Comment:
Control character

Reply:
This was an issue with my code fence logic and is fixed